students/ahwall/Lesson 02 Homework/FibLuc_Seq Exercise_HW2/FibLucSeq.py

```diff
@@ -0,0 +1,34 @@
+def fibinacci(x):
+    a = 0
+    b = 1
+    c = 0
+    if (x == 0):
+        print(x)
+    else:
+        for i in range(x):
+            a = b
+            b = c
+            c = b + a
+        print(c)
+
+
+def lucas(x):
+    a = 2
+    b = 1
+    if x == 0:
+        print(2)
+    elif x == 1:
+        print(1)
+    else:
+        for i in range(2, x + 1):
+            c = b + a
+            a = b
+            b = c
+        print(c)
+
+
+def sum_series(x, a = 0, b = 1):
```

Comment:
This isn't really what we were looking for, so hopefully you were able to follow along with the solution for sum_series in class. Let us know if you have any questions!